static/app/api/urlpatterns.generated.ts

```diff
@@ -0,0 +1,794 @@
+/**
+ * GENERATED FILE. Do not edit manually.
+ * To update it run `python3 -m tools.api_urls_to_typescript`
+ */
+
+type KnownApiUrls = [
+  '/',
+  '/',
+  '/accept-invite/$memberId/$token/',
```

Comment:
The urls in this list have `$` as the variable prefix, which aligns with `getApiUrl()`

But our routes.tsx file uses `:`. I think it'd be better switch the whole system over to that prefix, so routes and the static typing align.